Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/pageObjects/AddressesPage.java

```diff
@@ -0,0 +1,19 @@
+package com.griddynamics.qa.vikta.uitesting.sample.pageObjects;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AddressesPage {
+
+  @FindBy(css = "#tblAddresses > tbody > tr:last-child > td:nth-child(7)")
```

Comment:
let's not hardcode the index of element. it's better to create a selector that will allow us to pick up any address, e.g. by it's order on the list